doc/articles/uno-development/debugging-uno-ui.md

```diff
@@ -44,6 +46,15 @@ To debug Uno.UI code in the application, follow these steps (using `FrameworkEle
 7. You should hit the breakpoint, opening the `FrameworkElement.cs` file, and be able to see local variable values, etc.
 8. To revert to the original Uno.UI version from NuGet, simply navigate to the NuGet cache folder (`%USERPROFILE%\.nuget\packages`) and delete the `Uno.UI` folder within it. You may need to close Visual Studio first. The original version will be automatically restored the next time the application builds.
 
+### Troubleshooting
```

Comment:
- does this replace having to build `Uno.UI.Runtime.*`, or is that also worth mentioning?
 - also, what to do when you want to locally modify the public API surface on Skia/WASM (build Uno.UI as well as Uno.UI.* if I understand correctly, but I haven't tried yet)

Reply:
For uno.ui.runtime, no. I'll update the second in another PR.